packages/foreman/keycloak-httpd-client-install/keycloak-httpd-client-install.spec

```diff
@@ -4,16 +4,21 @@
 
 Name:           %{pypi_name}
 Version:        1.2.2
-Release:        1%{?dist}
+Release:        2%{?dist}
 Summary:        %{summary}
 
 License:        GPLv3
 URL:            https://github.com/latchset/keycloak-httpd-client-install
 Source0:        https://files.pythonhosted.org/packages/source/k/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
 BuildArch:      noarch
- 
+
+%if 0%{?rhel} == 7
```

Comment:
Usually, we don't do "if rhel", but have "with_pythonX" macros. see for example https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/client/python-apypie/python-apypie.spec

I wouldn't block on that, but strongly prefer ;)